src/content/blog/astro-5-beta.mdx

```diff
@@ -0,0 +1,159 @@
+---
+title: "Astro 5.0 Beta Release"
+description: "The first beta for Astro 5.0 beta is now available for you to try!"
+publishDate: "September 17, 2024"
+authors:
+  - erika
+coverImage: "/src/content/blog/_images/astro-5-beta/header_astro_5_beta.webp"
+socialImage: "/src/content/blog/_images/astro-5-beta/og_image_5_beta.webp"
+lang: "en"
+homepageLink:
+  title: "Astro 5.0 Beta"
+  subtitle: "Now available to try!"
+---
+
+import BlogSummary from "/src/components/BlogSummary.astro";
+
+<BlogSummary>
+  The first Astro 5.0 beta is now available! This release includes the now stable Astro Content Layer, our latest iteration on managing content in Astro projects, and server islands, a new way to mix static and personalized content on your site for the best of both worlds.
+
+	Read on to learn more about the new features and improvements in Astro 5.0 beta!
+</BlogSummary>
```

Comment:
`BlogSummary` doesn’t exist any more, so we’ll want to delete this or be in for a nasty surprise when updating branch:

```suggestion
**The first Astro 5.0 beta is now available! This release includes the now stable Astro Content Layer, our latest iteration on managing content in Astro projects, and server islands, a new way to mix static and personalized content on your site for the best of both worlds.**

**Read on to learn more about the new features and improvements in Astro 5.0 beta!**
```